vars.cmd

```diff
@@ -0,0 +1,21 @@
+@REM script for setting all the variables in order to run the ATH locally on windows
+@REM use config.groovy in the same folder if desired uncomment below
+@REM set CONFIG=%~dp0%config.groovy

```

Comment:
this script does not disable the `echo` output using `@echo off` so that consumers can see what is set.  hence things that should not be echoed start with `@`